src/python/director/cameracontrolpanel.py

```diff
@@ -5,6 +5,7 @@
 from director import cameracontrol
 from director import propertyset
 from director import pointpicker
+from director import roboturdf
```

Comment:
that's fine. it should be clear what it is being used to identify that the clicked object is a robot.

You might also want to remove RobotModelFollower as it is replaced by using the common tracker classes